paasta_tools/kubernetes_tools.py

```diff
@@ -1145,10 +1145,18 @@ def get_volume_claim_templates(self) -> Sequence[V1PersistentVolumeClaim]:
         ]
 
     def get_storage_class_name(self, volume: PersistentVolume) -> str:
-        storage_class_name = volume.get("storage_class_name", "ebs")
-        if storage_class_name not in SUPPORTED_STORAGE_CLASSES:
-            log.warning(f"storage class {storage_class_name} is not supported")
-            storage_class_name = "ebs"
+        try:
+            system_paasta_config = load_system_paasta_config()
```

Comment:
This would be a little cleaner if you caught the PaastaNotConfiguredError here. Then supported_storage_classes could be an empty list in that case, and then you only have to do the default once.